<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPresetOptionsDialog</class>
 <widget class="QDialog" name="SetPresetOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose presets</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Presets</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="Horizontal_radioButton">
        <property name="text">
         <string>Horizontal</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="Horizontal_reverse_radioButton">
        <property name="text">
         <string>Horizontal, reverse geocode range</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="Vertical_radioButton">
        <property name="text">
         <string>Vertical</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="Vertical_reverse_radioButton">
        <property name="text">
         <string>Vertical, reverse geocode range</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="Dataset_description_radioButton">
        <property name="text">
         <string>data set description</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="Parameter_statistic_radioButton">
        <property name="text">
         <string>parameter statistic</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Horizontal_radioButton</tabstop>
  <tabstop>Horizontal_reverse_radioButton</tabstop>
  <tabstop>Vertical_radioButton</tabstop>
  <tabstop>Vertical_reverse_radioButton</tabstop>
  <tabstop>Dataset_description_radioButton</tabstop>
  <tabstop>Parameter_statistic_radioButton</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
